edx_proctoring/api.py

```diff
@@ -3019,8 +3019,17 @@ def is_backend_dashboard_available(course_id):
         active_only=True
     )
     for exam in exams:
-        if get_backend_provider(name=exam.backend).has_dashboard:
-            return True
+        try:
+            if get_backend_provider(name=exam.backend).has_dashboard:
```

Comment:
We don't want this case to throw an error because we still want the instructor dashboard to load for courses that still have software secure chosen.